client/src/components/Home.svelte

```diff
@@ -283,6 +262,97 @@ onMount(() => {
                                 </div>
                             {/if}
                         </div>
+                        <div id="product-filter">
+                            <label
+                                    for="product-dropdown"
+                                    class="dropdown-label"
+                                    on:click={toggleProductDropdown}
+                            >
+                                Products:
+                                <span class="dropdown-arrow">{isProductDropdownOpen ? "▲" : "▼"}</span>
+                            </label>
+
+                            {#if isProductDropdownOpen}
+                                <div class="product-checkbox-group">
+                                    {#each products as product}
+                                        <div>
+                                            <input
+                                                    type="checkbox"
+                                                    id="product-{product}"
+                                                    value={product}
+                                                    bind:group={selectedProducts}
+                                            />
+                                            <label for="product-{product}">{product}</label>
+                                        </div>
+                                    {/each}
+                                </div>
+                            {/if}
+                        </div>
+                        <style>
+                            .dropdown-label {
```

Comment:
dont add style tags inside the html, there is a style section at the bottom of the file

Reply:
yeah we try to separate them for readability